### PR TITLE
Aggregate data and average

### PR DESCRIPTION
**Pull request by DibaKarimi**

### CODE REVIEW

File: server/api.js [hunk 1]
```diff
@@ -45,32 +27,54 @@ const getChannelHistory = async (channel, oldest, latest) => {
...
-	const userName = userInfo.ok
-		? userInfo.user.real_name
-		: res.status(400).json("The user not found");
-	result.userName = userName;
-	// console.log(userName);
-	const findReaction = (reactions) => {
-		return reactions.filter((reaction) => reaction.users.includes(userId));
+const getStat = async (userId, oldest,latest) => {
+	latest = latest == undefined? "" : latest;
+	console.log(latest);
```
Comment (Laura-TM) [inline server/api.js]: if you're happy this works, could you delete the console.logs?

File: server/api.js [hunk 2]
```diff
@@ -45,32 +27,54 @@ const getChannelHistory = async (channel, oldest, latest) => {
...
+	userName: "",
+	statistics: [],
+};
+const userInfo = await getUserInfo(userId);
+const userName = userInfo.ok
+	? userInfo.user.real_name
+	: res.status(400).json("The user not found");
+result.userName = userName;
+const now = new Date()/1000;
+const oldest =now - 60*60*7*24;
```
Comment (Laura-TM) [inline server/api.js]: would it be better to save this number as a const variable, e.g.

`const WEEK_IN_SECONDS = 60*60*7*24;`

then use it on lines 66 and 89 (where you have it as 604800000)

**Review by Laura-TM — APPROVED** [review]
Everything looks great :) I can test it out later. I made a couple of minor comments, but I will pre-approve so you can merge once you're happy

File: server/api.js (hunk 2 above)
Comment (DibaKarimi) [reply server/api.js]: Oh yes, It's a good idea. I'll do.
 Thank you, Laura.

**Pull request merged**